x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -228,6 +228,8 @@ export const importRulesRoute = (router: IRouter, config: LegacyServices['config
                         references,
                         note,
                         version,
```

Comment:
@FrankHassanabad I noticed that `lists` is absent from this branch, meaning that we wouldn't be able to, on import, overwrite a rule's lists. Happy to add that here if you want.

Reply:
Yeah you can here if you want or we can do another PR for it. But yes, it is missing here.

Reply:
Thank you for finding it.